unrejector/views/payment.erb

```diff
@@ -6,13 +6,82 @@
 <div class="section projects text-center">
   <h1 class="white open-sans">The Venue</h1>
   <div class="row">
-    <div class="small-10 small-centered large-4 large-centered columns">
-      <div class="panel">
-        <p></p>
-      </div>
+    <div class="small-10 small-centered large-6 large-centered columns">
+      <a href="http://www.galvanize.com/campuses/san-francisco-soma/">
+        <div class="panel">
+          <h1>Galvanize (SoMa)</h1>
+        </div>
+      </a>
     </div>
   </div>
 </div>
+<div class="section summary text-center">
+  <div class="row">
+    <div class="small-12 small-centered large-10 large-centered columns">
+      <p>
+        We're excited to let you know that your application has been accepted.
+      </p>
+      <p>
+        Camp is from 9:20am to 4:00pm Monday through Friday. If you can't
+        attend a session, make sure a parent lets a Hack Camp Facilitator know.
+        The curriculum is not hard, but can be fast-paced at times. To prevent
+        anyone from falling behind anyone with 2 unexcused absenses will be
+        dropped from the program. The deadline to pay your tuition is at 8pm on
+        Tuesday the 26th.
+      </p>
+      <p>
+        We look forward to seeing you on Monday (06/01) at Galvanize SF! If you
+        have any questions, please reach out to Zach Latta at zach@hackedu.us
+        or give him a call at 310-801-7309.
+      </p>
+    </div>
+  </div>
+</div>
+<div class="section red text-center">
+  <h1 class="open-sans"></h1>
+  <div class="row">
+    <div class="small-10 small-centered large-10 text-left columns">
+      <fieldset>
+        <legend>Personal Commitment</legend>
+        <input id="checkbox1" type="checkbox">
+        <label for="checkbox1">I will attend Hack Camp from 9:20am to 4:00pm
+          Monday through Friday.</label>
+      <br>
+      <input id="checkbox2" type="checkbox">
+      <label for="checkbox2"> If I cannot attend a session, a parent will let
+        Hack Camp facilitators know as soon as possible.</label>
+      <br>
+      <input id="checkbox3" type="checkbox">
+      <label for="checkbox3">I will be dropped from the program if I have 2
+        excused absences.</label>
+      <br>
+      <input id="checkbox4" type="checkbox">
+        <label for="checkbox4">I will bring a laptop.</label>
+      <br>
+      <input id="checkbox5" type="checkbox">
+      <label for="checkbox5">I understand the deadline to pay my tuition for
+        the program is 8pm, Tuesday the 26th.</label>
```

Comment:
Do these checkboxes disable the payment at the bottom unless all of them are checked, as discussed?